Implement a basic `Viscid Emission` mob TP skill

mrhappyasthma opened the pull request:
There is not a lot of info about this spell online. I based the implementation off of the existing `nepenthean_hum.lua`, which is also an AoE amnesia.

The difference being that this one is for a Cone (as reflected in the .sql).

Confirmed the animation by comparing with youtube videos from retail.

<!-- remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm: -->
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

mrhappyasthma committed:
Implement a basic 'Viscid Emission' mob TP skill

There is not a lot of info about this spell online. I based the implementation off of the existing `nepenthean_hum.lua`, which is also an AoE amnesia.

The difference being that this one is for a Cone (as reflected in the .sql).

Confirmed the animation by comparing with youtube videos from retail.

claywar commented on scripts/globals/mobskills/viscid_emission.lua:
```diff
@@ -0,0 +1,29 @@
+    return typeEffect
+end
+
+return mobskill_object
```
Could you please add a newline to the end of the file?

mrhappyasthma replied:
Sure. I'll apply this to my other pull request as well.

mrhappyasthma committed:
Add newline to end of file.

mrhappyasthma committed:
Removing unnecessary newline and inline local vars

Per the discussion on the other pull request, I'll just inline the locals that aren't reused and don't significantly increase line lengths.

mrhappyasthma commented:
Done as part of https://github.com/LandSandBoat/server/issues/202
